gpytorch/kernels/spectral_mixture_kernel.py

```diff
@@ -19,7 +19,8 @@ class SpectralMixtureKernel(Kernel):
 
         Unlike other kernels,
         * :attr:`ard_num_dums` **must equal** the number of dimensions of the data
-        * :attr:`batch_size` **must equal** the batch size of the data (1 if the data is not batched)
+        * :attr:`batch_shape` **must equal** the batch size of the data (1 if the data is not batched)
+        * :attr:`batch_shape` **cannot** contain more than one batch dimension.
```

Comment:
Is this necessary? We should check for it if so.

Reply:
Yes, I couldn't easily get the SM kernel supporting multi batch. I can do a second PR for it, but I also don't think it's mission critical to get 0.3.0 out.

Reply:
Yeah that's not mission critical, but then the kernel should check for that and throw an error.